R/readWrite.R

```diff
@@ -0,0 +1,96 @@
+#' Outputs a GTF file, transcript-count file, and gene-count file from bambu
+#' @title transcript to gene expression
+#' @param se a summarizedExperiment object from \code{\link{bambu}}
+#' @param path the destination of the output files (gtf, transcript counts, and gene counts)
+#' @return gtf a GTF dataframe
+#' @export
+write.bambu <- function(transcript_se,path){
+  if (missing(transcript_se) | missing(path)){
+    stop('Both summarizedExperiment object from bambu and the path for the output files are required.')
+  }else{
+    dir.create(path)
```

Comment:
test if the directory exists before creating it

Reply:
The output directory will only be created if it has not been existing.